backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteNonRootBranch.java

```diff
@@ -34,6 +37,9 @@
   private final List<IGitMacheteCommit> commits;
   private final SyncToRemoteStatus syncToRemoteStatus;
   private final SyncToParentStatus syncToParentStatus;
+  @Getter(AccessLevel.NONE)
```

Comment:
Nit: I guess the getter won't be generated even if `@Getter(AccessLevel.NONE)` is missing since there's already a nullary `getRemoteTrackingBranch` method in the class (?)